docs/iris/src/whatsnew/1.9.rst

```diff
@@ -0,0 +1,13 @@
+Iris 1.9 Bugs Fixed
+===================
+Iris 1.9 Features
```

Comment:
These sections aren't in the normal order. Can we control the order via the snippets or is it baked in to the build script? (Or is it random?)

If you don't already, it might be nice to be able to define the section titles and their order directly from the source snippets. For example, `01_Features\my_feature.rst`, `03_News\cf_unit_split.rst` ... where the directory names are sorted and the value after the underscore is the literal title. The same sorting could be applied to the snippets too.

Apologies if you'd handled some of this already!